config/initializers/spree.rb

```diff
@@ -16,3 +16,41 @@
 end
 
 Spree.user_class = "Spree::User"
+
+attachment_config = {
+  styles: {
+    mini:     "48x48>",
+    small:    "100x100>",
+    product:  "240x240>",
+    large:    "600x600>"
+
+  },
+  path:           "/:class/:id/:style/:basename.:extension",
+  default_url:    "/:class/:id/:style/:basename.:extension",
+  default_style:  "product"
+}
+
+if Rails.env.production?
+  # S3 image upload settings
+  attachment_config.merge! ({
+                             s3_credentials: {
+                               access_key_id:     ENV['AWS_ACCESS_KEY_ID'],
+                               secret_access_key: ENV['AWS_SECRET_ACCESS_KEY'],
+                               bucket:            ENV['S3_BUCKET_NAME'],
+                               s3_host_name:      ENV['S3_HOST_NAME']
+                             },
+                             storage:        :s3,
+                             s3_headers:     { "Cache-Control" => "max-age=31557600"  },
+                             s3_protocol:    "https",
+                             bucket:         ENV['S3_BUCKET_NAME'],
+                           })
+else
+  attachment_config.merge! ({
+                             storage:        :filesystem
+                           })
```

Comment:
__[rubocop]__
* Indent the right brace the same as the start of the line where the left brace is.